screens/FavMoreInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StyleSheet, View, Text, Image } from 'react-native';
+import styles from '../style/base';
+import PropRow from '../components/PropRow';
+
+export default function FavMoreInfo({ route, navigation }) {
+  const { favPokemon } = route.params;
+  const { loading } = route.params;
```

Comment:
we shouldn't be able to enter this screen when `loading` is true.

We don't even have to `disable` the button on the previous screen, because you don't display the button when `loading` is `true`, so it's gonna be always `false`.